src/components/ebay-carousel/index.js

```diff
@@ -286,37 +286,27 @@ function handleScrollEnd(scrollLeft) {
     }
 
     const { state } = this;
-    const { config, items } = state;
```

Comment:
This change was primarily to switch from checking for the closest `carousel-item` to instead looking for the closest group of items when `items-per-slide` is used.